src/accessControl/acp/index.js

```diff
@@ -385,14 +417,94 @@ export default class AcpAccessControlStrategy {
     this.#fetch = fetch;
   }
 
+  getInheritedPoliciesUrlsForResource() {
+    // get all policies that apply to this resource
+    const policies = acpv2.getPolicyUrlAll(this.#originalWithAcr);
+    // filter out owner and own resource policies
+    const namedPolicyUrls = namedPolicies.map(({ name }) => {
+      const containerUrl = getPolicyResourceUrl(
+        this.#originalWithAcr,
+        this.#policiesContainerUrl,
+        name
+      );
+      return getNamedPolicyUrl(containerUrl, name);
+    });
+    const inheritedPoliciesUrls = policies.filter(
+      (policyUrl) =>
+        !namedPolicyUrls.includes(policyUrl) &&
+        policyUrl.startsWith(this.#policiesContainerUrl)
```

Comment:
This is inline with what we discussed - hopefully we can find a better approach later, but this should work for now